File: T4.1.1_ModeladoRelacionesClasesPolimorfismo/Ecosystem/ecosystemcs/Eucariota.cs
public class Eucariota : Celula{
    private Boolean nucleo = true;

    public Boolean getNucleo() {
        return nucleo;
    }

    public void setNucleo(Boolean nucleo) {
        this.nucleo = nucleo;
    }

    public new String toString(){
        return base.toString() + "Existe nucleo? "+nucleo+"\n";
    }    
}

File: T4.1.1_ModeladoRelacionesClasesPolimorfismo/Ecosystem/Eucariota.java
public class Eucariota extends Celula{
    private boolean nucleo = true;

    public boolean getNucleo() {
        return nucleo;
    }

    public void setNucleo(boolean nucleo) {
        this.nucleo = nucleo;
    }

    public String toString(){
        return super.toString() + "Existe nucleo? "+nucleo+"\n";
    }    
}
